nixos/modules/services/computing/slurm/slurm.nix

```diff
@@ -0,0 +1,484 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.slurm;
+  # configuration file can be generated by http://slurm.schedmd.com/configurator.html
+  configFile = pkgs.writeText "slurm.conf" 
+    ''
+      ${optionalString (cfg.controlMachine != null) ''controlMachine=${cfg.controlMachine}''}
+      ${optionalString (cfg.controlAddr != null) ''controlAddr=${cfg.controlAddr}''}
+      ${optionalString (cfg.authType != null) ''authType=${cfg.authType}''}
+      ${optionalString (cfg.cacheGroups != null) ''cacheGroups=${cfg.cacheGroups}''}
+      ${optionalString (cfg.cryptoType != null) ''cryptoType=${cfg.cryptoType}''}
+      ${optionalString (cfg.mpiDefault != null) ''mpiDefault=${cfg.mpiDefault}''}
```

Comment:
Thank you, will remove unnecessary options. 
Can you tell me, who can I do this: 
I want to make an option like
services.slurm.partition.<name> = { nodes = lalalala; goo = bar; };
And translate it to
Partition=name nodes=lalalala foo=bar
So in configuration.nix I can write several partitions (like we have now, for example, with users).